packages/@okta/vuepress-site/guides/sign-into-spa/use-the-access-token/react/getaccesstoken.md

```diff
@@ -1,11 +1,11 @@
-Get the access token using the `getAccessToken()` method on the `Auth` object. Then, use this value to add an `Authorization` header to outgoing requests:
+Get the access token using the [getAccessToken()](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#authgetaccesstoken) method on the [Auth](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#auth) object object. Then, use this value to add an `Authorization` header to outgoing requests:
```

Comment:
Get the access token using the [getAccessToken()](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#authgetaccesstoken) method on the [Auth](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#auth) object. Then, use this value to add an `Authorization` header to outgoing requests: